packages/editor/src/editor/browser/data-class/variable-tree-data.ts

```diff
@@ -53,23 +53,27 @@ export const fullVariablePath = (row: Row<BrowserNode>, dontShowRootNode: boolea
 
   const parentPath = relevantParents.map(parent => parent.original.value).join('.');
 
-  if (row.original.value === 'Use entire Object' && dontShowRootNode) {
+  if (row.original.value === 'variable' && dontShowRootNode) {
     return '';
   }
   return parentPath ? `${parentPath}.${row.original.value}` : row.original.value;
 };
 
-export const rowToCreateData = (row: Row<BrowserNode>): CreateComponentData | undefined => {
+export const rowToCreateData = (
+  row: Row<BrowserNode>,
+  dontShowRootNode: boolean = false,
```

Comment:
normally better to not use a negation in the name. 
Why not switch to showRootNode?